core-bundle/src/Resources/views/Backend/be_route_path.html.twig

```diff
@@ -1,5 +1,5 @@
 {% trans_default_domain 'contao_tl_page' %}
-<div class="w50 widget">
+<div class="clr w50 widget">
```

Comment:
this fixes alignment issues if there is an error message in the alias field (e.g. if you enter a number)

Reply:
This should not be necessary. Can you post a screenshot please?

Reply:
I will revert this change, so I can merge the fix into the RC2. Feel free to open an issue if you can reproduce, then I‘ll take care of it.